.ci/vfio_jenkins_job_build.sh

```diff
@@ -0,0 +1,276 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# Run the .ci/jenkins_job_build.sh script in a VM
+# that supports VFIO, then run VFIO functional tests
+
+set -x
+
+cidir=$(dirname "$0")
+
+source /etc/os-release || source /usr/lib/os-release
+source "${cidir}/lib.sh"
+
+[ "${NAME}" == "Ubuntu" ] || die "Unsupported distro ${NAME}. Please use Ubuntu"
+
+vm_ip="127.0.15.1"
+vm_port="10022"
+data_dir="/tmp/kata-containers/vfio-test"
+ssh_key_file="${data_dir}/key"
+
+mkdir -p "${data_dir}"
+
+trap cleanup EXIT
+
+cleanup() {
+	sudo killall -9 qemu-system-x86_64
```

Comment:
Hardcoded architecture (here and below). Could you use `qemu-system-$(uname -i)` (or similar) instead?